Terminal.Gui/Core/Event.cs

```diff
@@ -8,6 +8,36 @@
 
 namespace Terminal.Gui {
 
+	/// <summary>
+	/// Manage the keys modifiers within a key event.
```

Comment:
Should be

/// Identifies the state of the "shift"-keys within a <see cref="KeyEvent"> event.